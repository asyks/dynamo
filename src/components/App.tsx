import React from 'react'

import EchoWrapper from './EchoWrapper'
import '../styles/App.css'


const App: React.FC = () => {
  return (
    <div className="App">
      <header className="App-header">
        dynamo: a network diagnostic tool
      </header>
      <EchoWrapper />
    </div>
  );
}

export default App;
