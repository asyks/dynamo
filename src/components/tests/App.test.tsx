import React from 'react'
import renderer from 'react-test-renderer'

import App from '../App'

describe("components/App", () => {
  test("nominally renders", () => {
    expect(renderer.create(<App />).toJSON()).toMatchSnapshot()
  })
})
