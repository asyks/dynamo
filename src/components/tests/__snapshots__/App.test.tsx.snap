// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/App nominally renders 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    dynamo: a network diagnostic tool
  </header>
  <div>
    <div>
      websocket echo
    </div>
    <button
      onClick={[Function]}
    >
      Connect
    </button>
    <input
      onChange={[Function]}
      type="text"
      value="HelloWebSocket!"
    />
    <button
      onClick={[Function]}
    >
      Send
    </button>
    <button
      onClick={[Function]}
    >
      Disconnect
    </button>
    <ul
      className="Web-socket-echo-log"
    />
  </div>
</div>
`;
